### PR TITLE
Upgrading RHVH hyperconverged host from Admin portal

### PR DESCRIPTION
**Pull request by rchikatw**
fails for first host

Upgrade is failing because ovit-ha-agent is trying to connect vdsm,
and if connection fails, it exits with a non-zero exit code.

after it fails systemd tries to restart the agent, and sincie the
agent wants vdsmd, it also starts vdsmd. This means that if you do:
systemctl stop vdmsd
and ovirt-ha-agent is in the middle of restart loop,
vdsmd service will start again. This is an impossible situation
so hence just before upgrade stopping the agent and
broker service.

Bug-Url: https://bugzilla.redhat.com/1979539
Signed-off-by: Ritesh Chikatwar <rchikatw@redhat.com>

### CODE REVIEW

**Comment by rchikatw** [comment]
ci add to whitelist

File: roles/cluster_upgrade/tasks/upgrade.yml [hunk 1]
```diff
@@ -84,6 +100,15 @@
     reboot_after_upgrade: "{{ reboot_after_upgrade }}"
     timeout: "{{ upgrade_timeout }}"
 
+- name: Start ovirt-ha-agent service
+  service:
+    name: ovirt-ha-agent
```
Comment (mnecas) [inline roles/cluster_upgrade/tasks/upgrade.yml:105]: Shouldn't we also start the `ovirt-ha-broker`?

Comment (rchikatw) [reply roles/cluster_upgrade/tasks/upgrade.yml:105]: No because ha-agent will start it. because ha-agent needs ha-broker

Comment (rchikatw) [reply roles/cluster_upgrade/tasks/upgrade.yml:105]: i verified it locally.

**Comment by dangel101** [comment]
LGTM

**Review by mnecas — CHANGES_REQUESTED** [review]
Noticed that there is a similar patch in the host upgrade [1], because we are calling host upgrade from cluster upgrade the code is not required here.
Could you please retry it locally without this change and only in host upgrade?
[1] https://gerrit.ovirt.org/#/c/ovirt-engine/+/115636/

**Review by mwperina — CHANGES_REQUESTED** [review]
-2

That should fixed inside host deploy and not in cluster upgrade role

**Comment by rchikatw** [comment]
True this is not required because host upgrade is being called internally. Hence closing this PR.

**Pull request closed without merging**